src/shapes.js

```diff
@@ -79,3 +79,8 @@ export const twitterCardShape = shape({
   app: twitterCardAppsShape,
   player: twitterCardPlayerShape,
 });
+
+export const alternateLinkShape = shape({
+  hreflang: string,
+  href: string,
+});
```

Comment:
[More attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#attr-alternate) are supported  with `<lin rel="alternate" />`. Let me know if I should update the PR to include more. My thinking is that `hreflang` and `href` are the two most relevant for SEO.